features/lorawan/lorastack/phy/loraphy_target.h

```diff
@@ -18,37 +18,50 @@
 #ifndef LORAPHY_TARGET
 #define LORAPHY_TARGET
 
+#define EU868 0
+#define AS923 1
+#define AU915 2
+#define CN470 3
+#define CN779 4
+#define EU433 5
+#define IN865 6
+#define KR920 7
+#define US915 8
+#define US915_HYBRID 9
+
```

Comment:
These end up permanently defined for any application code including `LoRaWANStack.h`. Again, maybe some concatenation could be done to allow short IDs in the JSON but not clutter C namespace. Could be done in future.